LeetCode/vDimple/String/806 Number of Lines To Write String/806 NumberOfLines.cpp

```diff
@@ -9,29 +9,24 @@ __declspec(dllexport) std::vector<int> numberOfLines(std::vector<int>& widths, s
 	{
 		sum += widths[int(s[i] - 'a')];
```

Comment:
Could we use here C++ style type casting ?

Reply:
Removed type casting.